### ISSUE TITLE
Is there any linux version?

### ISSUE THREAD

**Issue by aenciso** [smitthy09/claymore-v11.0-devfee-removed#2]
Thanks

**Comment by smitthy09** [comment]
no. and I propose to use windows for the simple reason that the drivers are more frequently updated and you can thus usually get a better hashrate.

**Comment by aenciso** [comment]
Thanks for your response, but ditched out windows long time ago. Drivers get updated frequently on Windows (can only speak for NVIDIA), only to add support for later games, it's barely related to mining which in that case CUDA would be more important though.

**Comment by smitthy09** [comment]
I rely exclusively on AMD. And here I must say the last few releases all upped my hash rate. So I am quite happy with windows... But thanks for the help!

**Comment by darkman7hr** [comment]
I bet I get better hash rates in slackware :D 
I won't even mention stability ;D
